front/public/locales/en/simulation.json

```
@@ -37,14 +37,19 @@
   "resetTimer": "Reset timer",
   "rotate": "Rotate the chart",
   "simplify": "Data simplification",
+  "speedSpaceChart": "Speed Space Chart",
   "speedSpaceSettings": {
-    "display": "Display",
-    "altitude": "Altitude",
-    "curves": "Curves",
-    "maxSpeed": "Line speed",
-    "slopes": "Slopes",
-    "electricalProfiles": "Electric traction",
-    "powerRestriction": "Power restriction"
+    "reticleInfos": "Reticle infos",
+    "energySource": "Energy source",
+    "tractionStatus": "Traction status",
+    "declivities": "Declivities",
```

Don't we use `slopes` for this ?

```suggestion
    "slopes": "Declivities",
```

---

no, we use slopes for declivities, curves and altitude, they will be implemented in the GEV later, I think we should have a proper term for each of them

---

We decided to use slopes to be consistent with backend but might be changed later because this word can be inconsistent with the data we manipulate.